src/maestro/agents/utils.py

```diff
@@ -0,0 +1,36 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright © 2025 IBM
+
+"""Common utility functions for agents."""
+
+from pathlib import Path
+from urllib.parse import urlparse
+from urllib.request import urlopen
+
+
+def is_url(text):
+    try:
+        result = urlparse(text)
+        return all([result.scheme, result.netloc])
+    except ValueError:
+        return False
+
+
+def is_file(text):
+    try:
+        return Path(text).exists()
+    except OSError:
+        return False
+
+
+def get_content(text):
+    if text is None:
+        return None
```

Comment:
Do we have list in `code` or `instructions`?

Reply:
a multiline value is read in as list, which only happens in our case for an in-line value and thus skip the url or path check

Reply:
passing in a list of string seems to only happen in tests when constructing the agent scheme manually instead of using the import, still leaving this here for compatibility